apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.html.ts

```diff
@@ -89,10 +102,30 @@ export function InstructorAIGenerateDrafts({
                   )}
                 </tbody>
               </table>
+
+              ${DestroyQuestionsModal(resLocals)}
             </div>
           </div>
         </main>
       </body>
     </html>
   `.toString();
 }
+
+function DestroyQuestionsModal(resLocals) {
+  return Modal({
+    id: 'destroyModal',
+    title: 'Delete all draft questions',
+    body: html` This will permamently and unrecoverably delete all of the draft questions. `,
```

Comment:
```suggestion
    body: 'This will permanently and unrecoverably delete all draft questions.',
```

Reply:
Done in 16aa671758e5d243976b2f58594e0032923eb6f8.